### PR TITLE
docs: updates the `sf` command on the `README.md`

### PR DESCRIPTION
**Pull request by renatoliveira**
This PR updates the commands used in the README to the newest `sf` command format.

Note: At L353 I added a bash line for the creation of an empty package.xml file because the deploy command requires a manifest.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,16 +112,16 @@ It's also important to implement a way to switch back to full deployment in case
 The plugin requires git command line on the running environment.
 
 **Node v16.20.0 or above is required**.
-To check if Salesforce CLI runs under a supported node version for SGD, run `sfdx --version`. You should see a node version above v.16.20.0 to use SGD.
+To check if Salesforce CLI runs under a supported node version for SGD, run `sf --version`. You should see a node version above v.16.20.0 to use SGD.
 
 If you encounter this issue whereas the node version is OK on the running environment, try to [install the Salesforce CLI via npm](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm#sfdx_setup_install_cli_npm) (`npm install sfdx-cli --global`).
```
Comment (scolladon) [inline README.md:117]: **remarks:** we should also change the sf installation

```suggestion
If you encounter this issue whereas the node version is OK on the running environment, try to [install the Salesforce CLI via npm](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm#sfdx_setup_install_cli_npm) (`npm install @salesforce/cli --global`).
```

**Review by scolladon — COMMENTED** [review]
Very nice !

We were planning to do this soon (starting next week probably) because everything will be deprecated in november.
Thank you very much.

I just noticed one thing
Maybe @mehdicherf you want to have a closer look as well

**Review by scolladon — APPROVED** [review]

**Comment by mehdicherf** [comment]
Thank you for the PR @renatoliveira .
That was something in our backlog, so it's awesome you took the time to do it 👍 

Regaring:
> Note: At L353 I added a bash line for the creation of an empty package.xml file because the deploy command requires a manifest.

Is it needed? 
As far I know, SGD **_already_** creates an empty `package.xml` file within the `destructiveChanges` folder. 
Why not using `destructiveChanges/package.xml` directly?
Is this file missing during your tests? Or did you face any issue in using it?

**Comment by scolladon** [comment]
Sorry guys, I kind of messed up this branch on renato repo...
I created another branch and a new PR with the same commits here #912

**Comment by renatoliveira** [comment]
> Thank you for the PR @renatoliveira . That was something in our backlog, so it's awesome you took the time to do it 👍
> 
> Regarding:
> 
> > Note: At L353 I added a bash line for the creation of an empty package.xml file because the deploy command requires a manifest.
> 
> Is it needed? As far as I know, SGD **_already_** creates an empty `package.xml` file within the `destructiveChanges` folder. Why not using `destructiveChanges/package.xml` directly? Is this file missing during your tests? Or did you face any issue in using it?

Ah, cool, I wasn't considering that when I made those changes. Good to know, so that line isn't really necessary. :)

**Pull request closed without merging**